internal/provider/inventory_data_source.go

```diff
@@ -79,7 +105,14 @@ func (d *InventoryDataSource) Read(ctx context.Context, req datasource.ReadReque
 		return
 	}
 
-	resourceURL := path.Join(d.client.getApiEndpoint(), "inventories", state.Id.String())
+	URI := path.Join(d.client.getApiEndpoint(), "inventories")
+	resourceURL, err := ReturnAAPNamedURL(state.Id, state.Name, state.OrganizationName, URI)
+	// resourceURL, err := state.ResourceUrlFromParameters(d)
```

Comment:
This function Read takes in a context but we dont watch it, any reason why? 

Also a commented out bit of code here 😄